### PR TITLE
fix(components): [image] imageViewer crossorigin configuration missing

### PR DESCRIPTION
**Pull request by declanchiu**
Please make sure these boxes are checked before submitting your PR, thank you!

- [] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by declanchiu** [comment]
fix  https://github.com/element-plus/element-plus/issues/17330

File: packages/components/image-viewer/src/image-viewer.vue [hunk 1]
```diff
@@ -127,6 +133,9 @@ const imgRefs = ref<HTMLImageElement[]>([])
 const scopeEventListener = effectScope()
 
 const loading = ref(true)
+const crossorigin = ref<'anonymous' | 'use-credentials' | undefined | ''>(
```
Comment (chenxch) [inline packages/components/image-viewer/src/image-viewer.vue:136]: This has already been declared in props and does not need to be declared again.

**Review by chenxch — APPROVED** [review]
LGTM. cc @element-plus/backers

**Review by btea — APPROVED** [review]
Thanks!

**Pull request merged**